centaur/src/main/scala/centaur/test/metadata/WorkflowFlatMetadata.scala

```diff
@@ -24,6 +24,11 @@ import scala.util.{Failure, Success, Try}
   */
 case class WorkflowFlatMetadata(value: Map[String, JsValue]) extends AnyVal {
 
+  def onlySubWorkflows = {
```

Comment:
If we use `diffson` to generate full-json diffs, we probably don't need this function anymore.